cider/exec/nextgen/jitlib/llvmjit/LLVMJITModule.cpp

```diff
@@ -19,22 +19,45 @@
  * under the License.
  */
 #include "exec/nextgen/jitlib/llvmjit/LLVMJITModule.h"
-
-#include <llvm/IR/Function.h>
-
+#include "exec/nextgen/jitlib/llvmjit/LLVMJITEngine.h"
 #include "exec/nextgen/jitlib/llvmjit/LLVMJITUtils.h"
 #include "util/Logger.h"
 
+#include <llvm/IR/Function.h>
+#include <llvm/IR/LLVMContext.h>
+#include <llvm/IRReader/IRReader.h>
+#include <llvm/Transforms/Utils/Cloning.h>
```

Comment:
Arrange header files as google cpp code style. Meanwhile, some header file may be redundant, remove them.

Reply:
fixed